content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -107,89 +99,123 @@ Once you have a Linux-based Kubernetes control-plane node you are ready to choos
-    ```
-    {{< note >}}
-    If you're using host-gateway use https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-host-gw.yml instead
-    {{< /note >}}
-
-    {{< note >}}
-If you're using a different interface rather than Ethernet (i.e. "Ethernet0 2") on the Windows nodes, you have to modify the line:
+### Joining a Windows worker node
 
-```powershell
-wins cli process run --path /k/flannel/setup.exe --args "--mode=overlay --interface=Ethernet"
-```
+{{< note >}}
+All code snippets in Windows sections are to be run in a PowerShell environment
+with elevated permissions (Administrator) on the Windows worker node unless otherwise noted.
+{{< /note >}}
 
-in the `flannel-host-gw.yml` or `flannel-overlay.yml` file and specify your interface accordingly.
+{{< tabs name="tab-windows-kubeadm-runtime-installation" >}}
 
-```bash
-# Example
-curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml | sed 's/Ethernet/Ethernet0 2/g' | kubectl apply -f -
-```
-    {{< /note >}}
+{{% tab name="CRI-containerD" %}}
 
+### Intro
 
+The following instructions are require HostProcess container support with Kubernetes 1.22+.  If you do not support HostProcess containers in you cluster you can install Flannel and kube-proxy as windows services directly on the host.  
 
-### Joining a Windows worker node
+The Before running any of these step the Windows node should have the following Windows Features installed: `Containers`,`Hyper-V`,`Hyper-V-PowerShell`.  These can be installed using the Powershell `Install-WindowsFeature` command.
 
 {{< note >}}
-All code snippets in Windows sections are to be run in a PowerShell environment
-with elevated permissions (Administrator) on the Windows worker node.
+The following instructions are the manual steps to configure nodes. You can use the [image-builder](https://image-builder.sigs.k8s.io/capi/windows/windows.html) used as part of the Cluster Api project implements the following along with other optimizations.
 {{< /note >}}
 
-{{< tabs name="tab-windows-kubeadm-runtime-installation" >}}
+#### Install containerD
 
-{{% tab name="CRI-containerD" %}}
+Follow the instructions for [installing the Containerd runtime on Windows using powershell](../../../setup/production-environment/container-runtimes.md#containerd)
 
-#### Install containerD
+#### Install kubelet
 
-```powershell
-curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/Install-Containerd.ps1
-.\Install-Containerd.ps1
+Kubeadm sets some flags via a environment file.  To support this it is common to use [nssm](https://nssm.cc/) to configure kubelet.  To install nssm do the following:
+
+``` powershell
+mkdir c:\k
+$arch = "win64"
+curl.exe -L https://k8stestinfrabinaries.blob.core.windows.net/nssm-mirror/nssm-2.24.zip -o nssm.zip
+tar.exe C c:\k\ -xvf .\nssm.zip --strip-components 2 */$arch/*.exe
 ```
```

Comment:
We should hyperlink to https://nssm.cc/download and recommend that readers follow the external tool's docs on how to download and install it. I believe that https://nssm.cc/release/nssm-2.24.zip is the canonical download URI for v2.24

We really should not be recommending to readers that they download and run tools from noncanonical sources. SIG Security have **opinions** on this and I believe those opinions are similar to “yeah, don't do that”.